subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AbstractRealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -24,7 +24,7 @@ import org.gradle.profiler.BuildMutator
 import org.gradle.profiler.ScenarioContext
 
 class AbstractRealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionPerformanceTest {
-    static final String SANTA_AGP_TARGET_VERSION = "3.6"
+    static final String SANTA_AGP_TARGET_VERSION = "4.1.2"
```

Comment:
Can we take this from some property where it's harder to forget to update it whenever a new stable AGP release happens?

Reply:
I moved the version properties to `AndroidTestProject`. So they are all now in one place.